.github/workflows/self-push-caller.yml

```diff
@@ -13,17 +14,8 @@ on:
 
 jobs:
```

Comment:
Can't comment above, bu I think we need a `workflow_dispatch:` under the `on:` above.

Reply:
`workflow_dispatch` is used to trigger a workflow manually:

_To manually trigger a workflow, use the workflow_dispatch event. You can manually trigger a workflow run using the GitHub API, GitHub CLI, or GitHub browser interface._
[(doc)](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)

And this is not in our original push-CI workflow file. I checked the current logic in my own dummy repo., and things work as expected (of course, without the huge chunk of test jobs in transformers.)

Let me know if you have other considerations for adding `workflow_dispatch`, thanks.